src/language/dafnyInstallation.ts

```diff
@@ -123,6 +129,32 @@ export class DafnyInstaller {
     }
   }
 
+  private async installFromSource() {
+    const { execSync } = require('child_process');
+    const execLog = (command: string) => {
+      this.writeStatus(`Executing: ${command}\n`);
+      execSync(command);
+    };
```

Comment:
When applying my above suggestion, you no longer need to inline this code. Again, the line-break should be redundant.

Reply:
What do you mean by "need to inline this code"?